Weakpoints & Proficiencies for Zombie-Adjacent Monsters

SolventMercury opened the pull request:
#### Summary
Content "Added Proficiencies & Weakpoints to Zombie-Adjacent Monsters"

#### Purpose of change
Builds on my last pull request, #64194. Essentially the same treatment, but for the "zombie-adjacent" monsters this time. This consists of the feral humans, fungal zombies, and enemies in the jabberwock.json file.

#### Describe the solution
Added appropriate data so that enemies had weakpoints and proficiencies defined for them as needed. Also made a few minor description tweaks.

#### Describe alternatives you've considered
N/A

#### Testing
Loaded into world with the new JSONs swapped in, made sure everything worked, tested a few enemies to ensure that changes stuck.

SolventMercury committed:
Weakpoints & Proficiencies for Zombie-Adjacent Monsters

SolventMercury committed:
Update data/json/monsters/fungus_zombie.json

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

SolventMercury committed:
Update data/json/monsters/fungus_zombie.json

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

SolventMercury committed:
Update data/json/monsters/fungus_zombie.json

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

hexagonrecursion commented on data/json/monsters/feral_humans.json:
```diff
@@ -737,6 +737,7 @@
     "melee_dice_sides": 3,
     "dodge": 1,
     "weakpoint_sets": [ "wps_humanoid_body" ],
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_syn_armored" ],
```
Description says "body-armor **wasn't** one of their priorities". Why prof_wp_syn_armored?

SolventMercury replied:
Ah you're right, I misparsed that lol.

SolventMercury replied:
Fixed

hexagonrecursion commented on data/json/monsters/jabberwock.json:
```diff
@@ -19,6 +19,7 @@
     "melee_dice": 2,
     "melee_dice_sides": 4,
     "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "families": [ "prof_intro_biology", "prof_wp_demihuman" ],
```
This is described as "an amalgamation". Maybe prof_wp_amalgamation?

SolventMercury replied:
No. Amalgamations as referred to with weakpoints are a very specific *thing*, like a zombie, as opposed to being a general category, like a humanoid. Specifically, they're what happens when creatures too small to zombify have their corpses shaped by the blob into weird and alien forms. All these monsters loooong predate the addition of amalgamations.

hexagonrecursion commented on data/json/monsters/jabberwock.json:
```diff
@@ -81,6 +83,7 @@
     "melee_dice": 4,
     "melee_dice_sides": 8,
     "melee_damage": [ { "damage_type": "cut", "amount": 3 } ],
+    "families": [ "prof_intro_biology", "prof_wp_demihuman" ],
```
prof_wp_amalgamation?

hexagonrecursion reviewed (changes requested):
Here are a couple nitpicks from me

SolventMercury committed:
Fixed Incorrect Proficiency

- Fixed the incorrect use of the Synthetic Armors proficiency on an enemy with no armor.
